KubeArmor/BPF/system_monitor.c

```diff
@@ -15,6 +15,12 @@
 #define asm_inline asm
 #endif
```

Comment:
Despite adding this in accordance with https://github.com/iovisor/bcc/issues/2546

Still getting the same errors when using the default version of clang provided in the `ubuntu-18.04` image in gh action, 
![image](https://user-images.githubusercontent.com/47106543/167141153-bd547fe9-6695-466f-9173-1f63884e54d5.png)
Ref https://github.com/kubearmor/KubeArmor/runs/6305329041?check_suite_focus=true

Any thoughts on why would this be happening?

It resolves if we upgrade the version of the clang which has the asm inline construct available, but still..

Reply:
what clang version does Ubuntu-18.04 use? what is the minimum version of clang to make KubeArmor work?

Reply:
what clang version does Ubuntu-18.04 use? what is the minimum version of clang to make KubeArmor work?

Reply:
> what clang version does Ubuntu-18.04 use? 

clang 9 in the GH Action env
clang 6 in the Vagrant env

> what is the minimum version of clang to make KubeArmor work?

It's not that straight forward
clang 6 works fine with kernel version 4.x.x
But with 5.x.x we need atleast clang 12